README.rst

```diff
@@ -85,6 +91,10 @@ Usage
                                       GDAL format driver documentation for more
                                       information.
 
+      --co, --profile NAME=VALUE      Driver specific creation options.See the
```

Comment:
```suggestion
      --co, --profile NAME=VALUE      Driver specific creation options. See the
```

Reply:
In fact, the formatting is due to a rasterio bug, but I'll fix the readme.